packages/contact-center/store/package.json

```diff
@@ -18,11 +18,11 @@
     "test:unit": "jest"
   },
   "dependencies": {
+    "@webex/plugin-cc": "3.5.0-wxcc.6",
     "mobx": "6.13.5",
     "react": "18.3.1",
     "react-dom": "18.3.1",
-    "typescript": "5.6.3",
-    "webex": "3.6.0-wxcc.5"
+    "typescript": "5.6.3"
```

Comment:
mark other packages as private

Reply:
We cant mark them private as there are dependencies in the cc-widgets, even if I keep them as devDependencies there packages are installed. So we would need to publish these as well. Another reason for not marking them private is that it would help us in giving out separate cdns for each widgets.